code/responsive design/style.css

```diff
@@ -0,0 +1,179 @@
+body {
+    margin: 0;
+}
+
+#profile {
+    background-color: #f9b8b6;
+    padding: 1em;
+    margin: 1em;
+    margin-bottom: 2em;
+}
+
+#profile > img {
+    outline: 0.3rem solid #fff;
+    margin: 0.5em 0;
+    width: 100%;
+    max-width: 600px;
+    height: auto;
+    margin: 1rem 0;
+}
+
+#profile > span {
+    font-size: 1.3em;
+    font-weight: 600;
+    padding-bottom: 1em;
+}
+
+.card {
+    border: 3px solid #006d9e;
+    margin: 0.2rem;
+    height: auto;
+}
+
+img {
+    width: 100%;
+    height: auto;
+}
+
+header {
+    background-color: #a7cddf;
+    font-size: 2em;
+    display: flex;
+    justify-content: center;
+}
+ #page-wrapper {
+    padding: 1rem;
+ }
+
+ @media (min-width: 500px) {  
+    #page-wrapper {  
+        display: flex;  
+        flex-wrap: wrap;  
+        justify-content: center;  
+    }  
+
+    #profile {  
+        flex: 1 1 300px; 
+        margin: 1em;  
+    }  
+
+    #gallery {  
+        display: flex;  
+        flex-wrap: wrap;  
+      flex-direction: column;
+        flex: 3 1 500px; 
+    }  
+
+    .card {  
+        flex: 1 1 200px;  
+        margin: 0.5em; 
+    }
+    .card {  
+        display: flex;  
+        flex-direction: row; 
+        height: auto;  
+        margin: 1px;
+    }  
+    
+    .card.right {  
+        flex-direction: row-reverse;  
+    }  
+    
+    .card img {  
+        width: 50%; 
+        height: auto; 
+    }  
+    
+    .card .photo-label {  
+        width: 50%; 
+        padding: 1em; 
+    }
+ }
+    
+ @media (min-width: 750px) {  
+    #page-wrapper {  
+        display: flex;
+        flex-wrap: wrap;  
```

Comment:
Remember that the styles from the first media query also apply to the second one, so you don't need to repeat anything between the two. These first two lines, for example, could be removed. There are more in this section that are repeated, see how many you can take out :)